en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,96 @@
+Choreo provides insightful observability features to drill down to the request level and confirm the root cause for anomalies you detect.
 
-!!! info
-    For explanation, we are using the sample data available in the product.
-    
-## Get started with the sample service
+This guide describes how you can detect anomalies and perform root cause analysis on a service deployed in Choreo. 
 
-1. Go to **Services** from the left navigation.
-2. Create a new service.
-    1. **If this is your first service**, 
     
-        1. Enter a name for the service and click **Create**. 
-           
-           ![Create a new service](../assets/img/observability/first-service-creation.png){.cInlineImage-full}
-           
-    2. **Else** (if you already have services created), 
+## Accessing observability data of a service
+
+Follow this procedure to try out the sample service:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Click **Observability**, and then click **Try Now**. This takes you to the **Observability** page. 
+3. Move the pointer to the right-end of the row for an active service, and then click **Observe**. If you do not have any active services, you can click **Try Sample**. 
+
+    ![Access the Observability view](../assets/img/observability/access-observability.png){.cInlineImage-full}
         
-        1. Click **+ Create** to start creating a new service.
-        2. Enter a name for the service and click **Create**.
-           
-           ![Create a new service](../assets/img/observability/service-creation.png){.cInlineImage-full}
+    You are directed to the sample service low-code view where you can see the following:
 
-3. From the left-hand menu, click **Observe**.
-4. On the right-hand bottom, click the **Sample Service** icon.
-   
-    ![Sample service popup](../assets/img/observability/sample-service-pop-up.png){.cInlineImage-full}
-   
-5. Click **Try Sample Service**.
+    - Low-code diagram
+    - Throughput graph   
+    - Latency graph
+    - Logs
+    - Diagnostic data
 
-    ![Try sample service](../assets/img/observability/try-sample-service.png){.cInlineImage-half}
+Here, you can detect anomalies and perform root cause analysis on the sample service.
     
-## Detect anomalies and perform root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
+## Detecting anomalies and performing root cause analysis
```

Comment:
We have not used the -ing form in the titles/headings in any of the other doc pages. Any specific reason to take this approach here? Should we consider changing?

Reply:
Addressed this.